docs/guides/Create-UWP-Controls.md

```diff
@@ -96,13 +96,7 @@ For example, let’s say you’ve set the TPMinV for you controls package to Win
     \lib\uap10.0\*
     \ref\uap10.0\*
 
-To enforce the appropriate TPMinV check, create an [MSBuild targets file](/visualstudio/msbuild/msbuild-targets) and package it under the build folder (replacing "your_assembly_name" with the name of your specific assembly):
-
-    \build
-      \uap10.0
-        your_assembly_name.targets
```

Comment:
curious - why did we remove the folder structure. Is that the instructions are self-sufficient that we don't need the graphical representation?

Reply:
Yes, I don't think the folder structure is necessarily helpful in this case (single-file placement), so I simplified.